src/MagnumPlugins/BasisImporter/BasisImporter.h

```diff
@@ -440,13 +440,23 @@ class MAGNUM_BASISIMPORTER_EXPORT BasisImporter: public AbstractImporter {
         /** @brief Target format */
         TargetFormat targetFormat() const;
 
+        /** @brief HDR target format */
+        TargetFormat targetFormatHdr() const;
```

Comment:
I'm deprecating direct plugin instantiation because it's next to impossible to test, causes maintenance issues with duplicated config etc.. It's in bdc2bfd7731884aff757e36ec6d106abe7e8ac8c, I just didn't push to master to not cause you headaches because I assumed you're already working on this in some way.

So ... I'm wondering if it makes sense to expose these two APIs at this point, probably not? Given they're not really callable in the usual case, and basically just add impossible-to-cover lines.

OTOH in that commit I'm keeping the `TargetFormat` as is, because it still feels like the best way to document the various formats, link to them, etc., but other than that it's only used internally, not settable or queryable by the user in any way.

(I also hope you're not relying on this direct instantiation anywhere in your codebase. IIRC it used to be the case many many years ago, but shouldn't be anymore.)

Reply:
Feel free to push to master, I can rebase on top of that

Reply:
Done.